modules/testing/shared/src/main/scala/io/circe/testing/ArbitraryInstances.scala

```diff
@@ -127,6 +127,34 @@ trait ArbitraryInstances extends ArbitraryJsonNumberTransformer with CogenInstan
   implicit def arbitraryAsArrayEncoder[A: Cogen]: Arbitrary[Encoder.AsArray[A]] = Arbitrary(
     Arbitrary.arbitrary[A => Vector[Json]].map(Encoder.AsArray.instance)
   )
+
+  // implicit def arbitraryACursor(implicit A: Arbitrary[Json]): Arbitrary[ACursor] = {
```

Comment:
Oops. I meant to remove this. 

I was playing around with generating arbitrary (successful) cursor traversals, but I didn't finish it.